src/sass/partials/_footer.scss

```diff
@@ -1,10 +1,15 @@
 .footer {
-  background-color: rgba(247, 247, 247, 0.95); //lite green
-  color: rgb(84, 84, 84); //dark green
+  background-color: rgba(247, 247, 247, 1); //lite grey
+  color: rgb(84, 84, 84); //dark grey
   text-align: center;
-  padding: 10px;
-  position: sticky;
-  bottom: 0;
+  padding: 30px;
+
+  @media screen and (max-width: $breakpoint-mobile) {
```

Comment:
Better to use "min-width: $breakpoint-tablet"

Reply:
Corrected